src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -185,7 +185,7 @@ object Read extends Logging {
     val r = if (isMapped) {
       Option(record.getStringAttribute("MD")) match {
         case Some(mdTagString) =>
-          val result = MappedRead(
+          val result = new MDTaggedRead(
```

Comment:
Since we now have the ability to represent mapped reads without MDtags, I think we should return such a read here if there is no mdtag string (on line 213)